### ISSUE TITLE
issue #4, August 2018

### ISSUE THREAD

**Issue by raidel-dev** [dalba1992/ringsidenews#4]

**Comment by raidel-dev** [comment]
Hi, A+.
Long time no see.
Hope you are well and your work goes well.
Hope to hear from you soon.
Kind regards.
Minh.

**Comment by raidel-dev** [comment]
Hello, Minh.
Long time, no see.
I am doing well, thanks.
Hope you are doing very well.
👍

**Comment by raidel-dev** [comment]
Hi, A+.
Skype is an public channel.
Since you mentioned about Github issue, here will be happened an issue.
Is there any way to delete messages in skype?
Hope to hear from you soon.
Regards.

**Comment by dalba1992** [comment]
Please delete all histories.

**Comment by raidel-dev** [comment]
Hi, I deleted, Minh.
And I sent request on Skype
Please accept it first
Or you can invite 4cf226c8d7f819cb

**Comment by raidel-dev** [comment]
I didn't receive your invitation yet.
My id is "+84904059757", name is Richard Nguyen.
We have to use this channel in general.
For quick call, we can use skype.
Thanks.
Minh.

**Comment by raidel-dev** [comment]
Can't you use freelancer?
If it is possible, please invite me "richardminh19@hotmail.com"

**Comment by raidel-dev** [comment]
I have no freelancer account.
Only my group 10 members are using 2 freelancer accounts.
So we cannot use freelancer account

**Comment by raidel-dev** [comment]
Never mind.
When I write "Okay ..." in skype, hope you to check this.
Mr Ju will go there with Hui soon.
How do you think I can communicate with boss?
What is convenient and quick?

**Comment by raidel-dev** [comment]
Okay.
Well, I will create a temp freelancer account in vm ware although I don't know when it will be suspended 

Thanks

**Comment by raidel-dev** [comment]
Okay.
Take care.

**Comment by raidel-dev** [comment]
Hello, Minh
https://www.freelancer.com/u/bidroid
Is this your account?
I am digipay2018
Thanks.

**Comment by raidel-dev** [comment]
Hi A+
I am Liu.
How are you ?
Long time no see.

I want to download some papers.
Please help me.

At first you can download and see the paper from "https://arxiv.org/abs/1706.01789"
The title is "Deep Alignment Network: A convolutional neural network for robust face alignment"
I want to download following reference papers of this paper.
[34, 10, 29, 31, 32, 21, 4, 35, 18, 16, 13, 7, 1]

One more paper
title: "Unconstrained correlation filters"
link: https://www.researchgate.net/publication/47157402_Unconstrained_correlation_filters

I want you to download and send me ASAP.

Thanks, Liu.

**Comment by raidel-dev** [comment]
Hi, Liu! 
Long time, no see.
I uploaded to (Liu2018-08-10)
Can you please send me the photo of your family?

Thanks and Warm Regards.

**Comment by raidel-dev** [comment]
Hi, A+.
This is our boss's request.
There are 5 face dataset including casia face, FDDB, wider face, celeba and aflw.
He needs explanation and a few sample images for each dataset.
I am looking forward to your quick reply.
Thanks and regards.
Minh.

**Comment by raidel-dev** [comment]
I posted a project named "Project for digipay2018" in freelancer.

**Comment by raidel-dev** [comment]
https://github.com/bciar/lab6
Liu(2018-08-10)
Ju(2018-08-10)
Please download all so that I can delete all data and clean repo.

**Comment by raidel-dev** [comment]
Did you bid freelacner?
Project for digipay2018

**Comment by raidel-dev** [comment]
topic:Optimal trade-off filters for noise robustness, sharpness of the correlation peak
url:https://www.osapublishing.org/ol/fulltext.cfm?uri=ol-16-11-829
topic:Minimum-variance synthetic discriminant functions
url:https://www.researchgate.net/publication/249331221_Minimum-variance_synthetic_discriminant_functions
topic:Minimum average correlation energy filters
url:https://www.researchgate.net/publication/44618448_Minimum_average_correlation_energy_filters

**Comment by raidel-dev** [comment]
Please remove all data in lab6 repository.

Speaking about Ju's request, he needs how many images each dataset has, how much image resolution is and a few images for each dataset.
I think you can get such data in the download page of the dataset.
Sorry for cutting your time.
Take care.

**Comment by raidel-dev** [comment]
Hello, Bro.
I uploaded your data.
The first two links are not downloadable so I searched related documents and downloaded.
Hope it would be helpful.
same link: lab6/minh8.13

For the dataset, It takes about 4~5 hours for each dataset 
becuase it's big data, and during download, it fails continuously. 
but I am doing my best to download now. so will upload one by one as soon as it is downloaded.

Thanks.

**Comment by raidel-dev** [comment]
Hi, Bro.
Thanks for your effort.
Speaking about dataset, I don't need total data, but a few images for each.
If impossible, the details about dataset is okay, i.e. image resolution, number of images, etc.
Couldn't you find them in download page?
I don't want you to spend much time for that.
Hope to hear from you soon.
Kind regards.

**Comment by raidel-dev** [comment]
Okay, I extracted all necessary description for each dataset download page.
And I uploaded dataset for FDDB in our lab repo.

FDDB: Welcome to the Face Detection Data Set and Benchmark (FDDB), a data set of face regions designed for studying the problem of unconstrained face detection. This data set contains the annotations for 5171 faces in a set of 2845 images taken from the Faces in the Wild data set. More details can be found in the technical report below. 

 casia face:
 CASIA Face Image Database Version 5.0 (or CASIA-FaceV5) contains 2,500 color facial images of 500 subjects. The face images of CASIA-FaceV5 are captured using Logitech USB camera in one session. The volunteers of CASIA-FaceV5 include graduate students, workers, waiters, etc. All face images are 16 bit color BMP files and the image resolution is 640*480. Typical intra-class variations include illumination, pose, expression, eye-glasses, imaging distance, etc.
   The images of CASIA-FaceV5 are stored as:
$root path$/ YYY/YYY_X.bmp
YYY: the unique identifier of the subject in the subset
X: the index of image for each subject

wider face:
WIDER FACE dataset is a face detection benchmark dataset, of which images are selected from the publicly available WIDER dataset. We choose 32,203 images and label 393,703 faces with a high degree of variability in scale, pose and occlusion as depicted in the sample images. WIDER FACE dataset is organized based on 61 event classes. For each event class, we randomly select 40%/10%/50% data as training, validation and testing sets. We adopt the same evaluation metric employed in the PASCAL VOC dataset. Similar to MALF and Caltech datasets, we do not release bounding box ground truth for the test images. Users are required to submit final prediction files, which we shall proceed to evaluate.

celeba:
CelebFaces Attributes Dataset (CelebA) is a large-scale face attributes dataset with more than 200K celebrity images, each with 40 attribute annotations. The images in this dataset cover large pose variations and background clutter. CelebA has large diversities, large quantities, and rich annotations, including
10,177 number of identities,
202,599 number of face images, and
5 landmark locations, 40 binary attributes annotations per image.
The dataset can be employed as the training and test sets for the following computer vision tasks: face attribute recognition, face detection, and landmark (or facial part) localization.

aflw
The motivation for the AFLW database is the need for a large-scale, multi-view, real-world face database with annotated facial features. We gathered the images on Flickr using a wide range of face relevant tags (e.g., face, mugshot, profile face). The downloaded set of images was manually scanned for images containing faces. The key data and most important properties of the database are:
The database contains about 25k annotated faces in real-world images. Of these faces 59% are tagged as female, 41% are tagged as male (updated); some images contain multiple faces. No rescaling or cropping has been performed. Most of the images are color although some of them gray-scale.
In total AFLW contains roughly 380k manually annotated facial landmarks of a 21 point markup. The facial landmarks are annotated upon visibility. So no annotation is present if a facial landmark, e.g., left ear lobe, is not visible.
A wide range of natural face poses is captured The database is not limited to frontal or near frontal faces.
Additional to the landmark annotation the database provides face rectangles and ellipses. The ellipses are compatible with the FDDB protocol. Further, we include the coarse head pose obtained by fitting a mean 3D face with the POSIT algorithm.
A rich set of tools to work with the annotations is provided, e.g., a database backend that enables to import other face collections and annotation types. Also a graphical user interface is provided that enables to view and manipulate the annotations.
Due to the nature of the database and the comprehensive annotation we think it is well suited to train and test algorithms for:

**Comment by raidel-dev** [comment]
Thanks, Bro.

**Comment by raidel-dev** [comment]
I invited you to skype.
I am allowed to have chat with you in skype.
However, we can say about development.
To use skype is good for call and simple discussion, I think.
I hope you not to mention about github issue on skype.
When saying "Good...", let's check github.
What do you think?
Kind regards.
Minh.

**Comment by raidel-dev** [comment]
Okay, Good idea~

**Comment by raidel-dev** [comment]
Boss is leaving tomorrow.

**Comment by raidel-dev** [comment]
I see. thanks

**Comment by raidel-dev** [comment]
come to freelancer
